### ISSUE TITLE
How do I run this on my spreadsheet?

### ISSUE THREAD

**Issue by noam8080** [nuket/google-sheets-geocoding-macro#13]
It seems like Google changed it a little. I dont see an "Edit script" item in the Tools menu bar on Google Sheets.
I'm new to this. Can someone explain how I run it on my doc?
Thanks

**Comment by nuket** [comment]
Are you looking at a Sheet of your own or the one that I linked?

If it's your own Sheet, the Tools -> Edit Script menu should show up. This is a feature of Google Sheets, for logged in users. With your own sheet, you copy the code in the Code.gs into the App Script project.

I'll add some screenshots to the README.md to describe this.

For the linked Sheet, there are three things that can happen depending on how Google Docs sees you:

**Owner** -- the Tools -> Edit Script menu shows up only for me.
**Logged In** -- the Geocode menu should show up. If you're not logged in, the Geocode menu will not show up. It's all about billing.
**Anonymous** -- the Geocode menu will not show up. I just tried this with an Incognito Mode window.
